docs/contributing/forem.md

```diff
@@ -0,0 +1,180 @@
+   `git commit -m 'Add some feature'`
+4. Push to the branch: `git push -u origin USERNAME/some-new-feature-1234`
+5. [Create a pull request](https://docs.dev.to/getting-started/pull-request/)
+   for your branch. 🎉
+
+## Contribution guidelines
+
+### Create an issue
+
+Nobody's perfect. Something doesn't work? Something could be better? Check to
+see if the issue already exists, and if it does, leave a comment to get our
+attention! If the issue doesn't already exist, feel free to create a new one. A
+core team member will triage incoming issues.
+
+_Please note: core team members may update the title of an issue to reflect the
+discussion._
+
+### Please include tests
+
+Some areas of the project could use updated tests, and new features should
+always include test coverage. Please give our
+[testing guide](https://docs.dev.to/tests/) a read!
+
+### Code quality
+
+We use [CodeClimate](https://codeclimate.com/) to find code smells. If a pull
+request contains code smells, we might recommend a refactor before merging. We
+like readable code, and encourage DRY when it's reasonable!
+
+More importantly, we avoid
+[wrong abstractions](https://www.sandimetz.com/blog/2016/1/20/the-wrong-abstraction).
+Code quality tools are not perfect, so don't obsess over your CodeClimate score.
+
+### Please use inclusive language
+
+Inclusion and respect are core tenets of our
+[code of conduct](https://dev.to/code-of-conduct). We expect thoughtful language
+all the way down to the code. Some technical metaphors are alienating or
+triggering. We ask that contributors go the extra mile to submit code which is
+inclusive in nature.
+
+If you unintentionally use language deemed harmful, there is no shame. We will
+work together to find a better alternative. Being thoughtful about language also
+encourages more thoughtful code!
+
+### Create a pull request
+
+- Try to keep the pull requests small. A pull request should try its very best
+  to address only a single concern.
+- For work in progress Pull Requests, please use the
+  [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/)
+  feature.
+- Make sure all tests pass and add additional tests for the code you submit.
+  [More info here](https://docs.dev.to/tests/).
+- Document your reasoning behind the changes. Explain why you wrote the code in
+  the way you did. The code should explain what it does.
+- If there's an existing issue, reference to it by adding something like
+  `References/Closes/Fixes/Resolves #123`, where 123 is the issue number.
+  [More info here](https://github.com/blog/1506-closing-issues-via-pull-requests).
+- Please fill out the PR Template when making a PR.
+- All commits in a pull request will be squashed when merged.
+
+_Please note: a core team member may close your PR if it has gone stale or if we
+don't plan to merge the code._
+
+### Pull request reviews
+
+All community pull requests are reviewed by our core team.
+
+- All contributors must sign the CLA
```

Comment:
```suggestion
- All contributors must sign the CLA.
```

Reply:
Actually we should just add `.` at the end of all of the sentences in this bulleted list; looks like you did that for the preceding lists and I quite like that style (plus, consistency!).